src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -1357,7 +1356,6 @@ internal sealed class PlainUnboundLambdaState : UnboundLambdaState
             _isStatic = isStatic;
         }
 
-        public override bool HasNames { get { return !_parameterNames.IsDefault; } }
```

Comment:
so this member was just totally redundant in practice?

Reply:
Yup, they have the same meaning. We had `HasNames`, `HasSignature` and I was adding yet another one...